WELCOME.md

```diff
@@ -0,0 +1,47 @@
+👋 Hello @new_user_name and welcome to the For the Horde community! We're so happy you're here! The purpose of this community is to give you a space to connect with other Minnesota-based tech and tech-adjacent folks. Want to be successful here? Give more than you take.
+
+✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to [http://git.io/v0d6b](http://git.io/v0d6b).
+
+Need to report Code of Conduct violation? The admins are listed at the bottom of the Code of Conduct. Snap a screenshot, report it.
+
+🚴🏿‍♀️🕺🏻👩‍💻😻⚾️✈️📺 Second, you've automagically been added to the #introductions channel. Please introduce yourself to the community there and share something fun about yourself. Where you're from, what you're working on, what your most peculiar hobby is, what your favorite color is, etc.. We want to get to know you! Also, there's probably a channel for that.
+
+📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~600 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
+
+_Popular channels:_
+
+- #frontend - talk about your javascript
+- #devs-n-ops - do not talk about your javascript
+- #cohordeworking - when, where, why, who
+- #business - general business topics
+- #mental-health - talk about it
+- #gigs - the ONLY place you can post/ask for jobs/gigs. They should be 100% available to Minnesota with no filter criteria, ala “might be interested in remote” Many people have notifications on this channel, so 🧵any conversations beyond the initial post.
+
+_Fun, small channels:_
+
+- #art-hackers
+- #biking
+- #awww
+- #pets
+- #sports
+- #outdoors
+
+🦠 And then there’s COVID-19, the big 🐘 in the room. #general is a COVID-19 free zone. This is a **strict** policy and we will actively police it. While we understand it’s a hard time for everyone, you can’t leave #general and we want people to have a refuge as it is impacting many of us indirectly or directly.
+
+SO with that:
+#covid19-discuss - general discussion of COVID-19 related topics
+#covid19-reference - fact based information sharing (links only, move discussions to #covid19-discuss)
+#covid19-memes - the ONLY place to allow covid related memes (that also adhere to the Code of Conduct)
+
+:spam: Spam posts, sales-y posts, self-promoting without community participation is not allowed. Admins will ask posts like this to be removed or moved to a different channel. If not done by the original poster within 15 min, we will do it for you. Don't make us do it for you.
```

Comment:
```suggestion
:spam: Spam posts, sales-y posts, self-promoting without community participation is not allowed. For the Horde is a community to participate in, not an audience to broadcast to. Admins will ask posts like this to be removed, especially in #general. If not removed promptly, we will remove it. If you're really sure the post benefits the audience more than yourself, be sure to put it in the most relevant channel.
```
Riffing here, which is kind of awkward for prose in github. Aims I had with these changes:
* remove specific time limit (puts admins on a long clock monitoring the situation, and promises bad actors their spam will live for at least 15 minutes)
* introduce the community vs audience concept (I've become really attached to that language because I think it succinctly says a lot, in a conversation where "spam" and "promotion" can be hotly debated
* try to set a criteria for whether something is too spammy/promotional (does it benefit the audience more than the poster)

Reply:
yes, i love this!